files/ansible-test-branch.txt

```diff
@@ -1 +1 @@
-devel
+stable-2.16
```

Comment:
Create a new `stable-2.16` branch from `main` and update this PR to target that new branch.

A similar set of changes will need to be applied in another PR against `main`, which will still reference `devel`.